### PR TITLE
[SPARK-6850] [SparkR] use one partition when we need to compare the whole result

### PR DESCRIPTION
**Pull request by davies**

### CODE REVIEW

**Comment by davies** [comment]
cc @shivaram

File: R/pkg/inst/tests/test_binaryFile.R [hunk 1]
```diff
@@ -27,7 +27,7 @@ test_that("saveAsObjectFile()/objectFile() following textFile() works", {
   fileName2 <- tempfile(pattern="spark-test", fileext=".tmp")
   writeLines(mockFile, fileName1)
 
-  rdd <- textFile(sc, fileName1)
+  rdd <- textFile(sc, fileName1, 1)
```
Comment (shivaram) [inline R/pkg/inst/tests/test_binaryFile.R]: Not related to this PR but the `1` vs. `1L` thing (i.e. numeric vs. int) should also be standardized across all functions (they should accept numeric)

**Comment by shivaram** [comment]
LGTM - pending jenkins

File: R/pkg/inst/tests/test_binaryFile.R (hunk 1 above)
Comment (davies) [inline R/pkg/inst/tests/test_binaryFile.R]: created https://issues.apache.org/jira/browse/SPARK-6852

**Comment by shivaram** [comment]
Thanks @davies . Merging this

**Pull request closed without merging**